test/testrecipes/test-terraform-recipes/README.md

```diff
@@ -30,12 +30,12 @@ Then you can access the recipe download at:
 http://localhost:8999/<recipe_name>.zip
 ```
 
----- 
+---
```

Comment:
What does this update do?

Reply:
This is a refactoring of the README.md. That extra `-` was unnecessary and "shift + opt + F" in VS Code cleans it up.